spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/DnnBase.scala

```diff
@@ -69,6 +69,8 @@ trait MklDnnModule extends MklDnnModuleHelper {
   private[mkldnn] def gradOutputFormats(): Array[MemoryData]
 
   private[mkldnn] def gradOutputWeightFormats(): Array[MemoryData]
+
+  def setQuantizeFlag(value: Boolean): this.type
```

Comment:
`setQuantization` ?  no need `flag`

Reply:
Yes.

Reply:
@jason-dai  One driving motivation is we plan to decouple layer implementation from underlying engine and make layer unified, this is planned along with mkl-dnn upgrade soon. Also it should be better to keep format only and runtime engine irrelevant layer definition.  That's why we gave up the option to save DNN defined models